source/codegen/templates/proto_helpers.mako

```diff
@@ -30,16 +30,22 @@ enum ${service_class_prefix}Attributes {
 <%
   enum = enums[enum_name]
   enum_value_prefix = enum.get("enum-value-prefix", common_helpers.pascal_to_snake(enum_name).upper())
+  is_mapped_enum = enum.get("generate-mappings", False)
+  enum_name = f"{enum_name}Mapped" if (is_mapped_enum and not enum_name.endswith("Mapped")) else enum_name
   allow_alias = enum.get("allow-alias", proto_helpers.should_allow_alias(enum))
```

Comment:
I feel the suffix string for mapped name `{enum_name}Mapped`  cold be replaced with `{enum_name}+MAPPED_ENUM_SUFFIX`  with MAPPED_ENUM_SUFFIX = "Mapped".

Reply:
We have used `Mapped` keyword in lot of places. I think using formatted string makes it concise and more readable than using a variable.